### PR TITLE
add a new blog of Kubernetes-multicluster-KubeSphere

### PR DESCRIPTION
**Pull request by faweizhao26**
Signed-off-by: faweizhao26 <faweizhao@kubesphere.io>

### CODE REVIEW

File: content/zh/blogs/Kubernetes-multicluster-KubeSphere.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: '混合云下的 Kubernetes 多集群管理与应用部署'
+tag: 'KubeSphere, Kubernetes, 多集群管理'
+keywords: 'KKubeSphere, Kubernetes, 多集群管理, Kubefed'
```
Comment (FeynmanZhou) [inline content/zh/blogs/Kubernetes-multicluster-KubeSphere.md:4]: KKubeSphere => KubeSphere

File: content/zh/blogs/Kubernetes-multicluster-KubeSphere.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+description: '本文介绍了 Kubernetes 社区多集群方向的发展历程以及已有的多集群解决方案，分享在混合云的场景下, KubeSphere 如何基于 Kubefed 统一应用的分发与部署，以达到跨 region 的多活/容灾等目的。同时探讨未来多集群领域可能迈向的去中心化的架构。'
+createTime: '2021-05-26'
+author: ' 李宇'
+snapshot: 'https://pek3b.qingstor.com/kubesphere-community/images/Kubernetes-multicluster-KubeSphere-banner.jpg'
+---
+
+
+> 本文是上海站 Meetup 中讲师李宇根据其分享内容梳理成的文章。本文介绍了 Kubernetes 社区多集群方向的发展历程以及已有的多集群解决方案，分享在混合云的场景下, KubeSphere 如何基于 Kubefed 统一应用的分发与部署，以达到跨 region 的多活/容灾等目的。同时探讨未来多集群领域可能迈向的去中心化的架构。
```
Comment (FeynmanZhou) [inline content/zh/blogs/Kubernetes-multicluster-KubeSphere.md:12]: Do you consider adding the link of the video recording at the beginning?

Comment (faweizhao26) [reply content/zh/blogs/Kubernetes-multicluster-KubeSphere.md:12]: Done.

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**